### ISSUE TITLE
Specifying //JAVA shows wrong version in JBang panel

### ISSUE THREAD

**Issue by quintesse** [jbangdev/jbang-idea#56]
If I specify `//JAVA 17` in the file but my default Java is 13 then the JBang panel in the IDE shows 13, not 17.

**Comment by linux-china** [comment]
Yes. it's a bug, and  I fixed it already.  Still a question here, it's some expensive to call jbang command line to refresh JBang script info when file modified, and you should click refresh button to refresh script info.  It is alike Maven refresh in IDEA, after pom.xml modified, and you should click refresh button by yourself.

**Comment by maxandersen** [comment]
This should be addressed by the rewrite in PR #159 (version 0.100.0). Please try it out and reopen if the issue persists — feedback welcome!
